src/eval-frame/components/panes/declared-variable.jsx

```diff
@@ -5,15 +5,14 @@ import ValueRenderer from "../reps/value-renderer";
 
 export class DeclaredVariable extends React.Component {
   static propTypes = {
-    value: PropTypes.any, // eslint-disable-line react/forbid-prop-types
     varName: PropTypes.string
   };
   render() {
     return (
       <div className="declared-variable">
         <div className="declared-variable-name">{this.props.varName} = </div>
         <div className="declared-variable-value">
-          <ValueRenderer valueToRender={this.props.value} />
+          <ValueRenderer windowValue valueKey={this.props.varName} />
```

Comment:
`windowValue` flag tells the ValueRendered to look on the window object for the value it wants